bigquery/google/cloud/bigquery/job.py

```diff
@@ -2483,24 +2483,73 @@ class QueryPlanEntry(object):
     :type entry_id: int
     :param entry_id: ID of the entry
 
+    :type start_ms: int
```

Comment:
Howdy. Not to dump on you for docstrings again, but any chance we could update these to Google-style, too? Would be good to do now since we are doubling the number of properties.